experimental/trusted_shuffler_example/README.md

```diff
@@ -0,0 +1,52 @@
+# Trusted Shuffler Example
+
+This directory contains a Trusted Shuffler example application.
+
+## Components
+
+### Backend
+
+Backend server for the Trusted Shuffler example.
+
+Backend is used in tests/experiments and is represented as a simple HTTP echo
+service.
+
+Build and run the Backend with the following command:
+
+```bash
+./experimental/trusted_shuffler_example/scripts/run_backend
+```
+
+Backend code is in the `backend` directory.
+
+### Client
+
+Client that connects to the Trusted Shuffler and sends a number of requests via
+HTTP.
+
+Build and run the Client with the following command:
+
+```bash
+./experimental/trusted_shuffler_example/scripts/run_client
+```
+
+Client code is in the `client` directory.
+
```

Comment:
Maybe we could add one sentence on what we will have in `common`?

Reply:
Done